src/Libraries/Tesellation/Tessellation.csproj

```diff
@@ -22,6 +22,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+	<DocumentationFile>..\..\..\bin\AnyCPU\Debug\Tessellation.XML</DocumentationFile>
```

Comment:
XML needs to be built at en-US folder. Look at other projects how this is handled. Don't hardcode the path, use configuration properties such as $(OutputPath)